lib/please_cli/please_cli/terraform_route53_config.py

```diff
@@ -122,16 +131,62 @@ def cmd(channel):
                     domain = domain.lstrip('http')
                     domain = domain.lstrip('://')
 
-                if project_target == 'HEROKU':
-                    click.echo(HEROKU_TEMPLATE % dict(
-                        name=to_route53_name(project_id, channel),
-                        domain=domain,
-                        dns=project[channel]['dns'],
-                    ))
+                def heroku(a, b):
+
+                    if project_target == 'HEROKU' and a == channel and 'shipit-' not in project_id:
+                        b.append(HEROKU_TEMPLATE % dict(
+                            name=to_route53_name(project_id, channel),
+                            domain=domain,
+                            dns=project[channel]['dns'],
+                         ))
+
+                def shipit(a, b):
+
+                    if project_target == 'HEROKU' and a == channel and 'shipit-' in project_id:
+                        b.append(HEROKU_TEMPLATE % dict(
```

Comment:
it would be better to split the display logic to section below where you are using click.echo to print on console screen